netlify.toml

```diff
@@ -21,3 +21,15 @@
     # Set which environment variables to use for the approle login
     secretId = 'VAULT_SECRET_ID'
     roleId = 'VAULT_ROLE_ID'
+
+# HEADERS
+# ##############################################################################
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Access-Control-Allow-Origin = "*"
+    Content-Security-Policy = "default-src https: 'unsafe-inline' 'unsafe-eval'; form-action https:; img-src https: data:; frame-ancestors https://app.storyblok.com"
```

Comment:
Add `frame-ancestors` here to replace the older `X-Frame-Options = "DENY"`
Support for this is not an issue anymore:
https://caniuse.com/?search=frame%20ancestors

Reply:
Also, recommended by Storyblok here:
https://www.storyblok.com/faq/setting-content-security-policy-http-header

Reply:
You may want to consider setting these headers on the `/editor` route vs the whole site.

Reply:
@sherakama Yeah Storyblok got rid of the /editor path in the most updated version of their recommended setup. I actually have other question for you about the visual editor thing (other than a lot of issues that Storyblok needs to fix - seems that way for other frameworks too (next.js etc) from what I've seen on Discord). Will talk to you in our 1:1 tomorrow :)